### PR TITLE
Move docs creation to CI

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR removes the `docs` folder and make some adjustments in order to configure the Jenkins pipeline to build and push the compiled docs. This is better for mantainabilty as developers will only have to update the source content.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -58,25 +61,25 @@ pipeline {
...
-        }
-      }
-      steps {
-        script {
-          sh "docker build -f docker/Dockerfile-test -t love-frontend-test  ."
-          sh "docker run love-frontend-test"
-        }
-      }
-    }
+    // stage("Run tests") {
```
Comment (wvreeven) [inline Jenkinsfile:64]: In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.

Comment (wvreeven) [reply Jenkinsfile:64]: Also, are you sure you want to disable running the unit tests?

**Review by wvreeven — CHANGES_REQUESTED** [review]
What is the reason for disabling the unit tests? Please try to find a way to keep running them.

File: Jenkinsfile (hunk 1 above)
Comment (sebastian-aranda) [reply Jenkinsfile:64]: Sorry, this was intended for testing a further step. I'll make the change. Thanks!

**Review by wvreeven — APPROVED** [review]
Looks good to me. Thanks for the additional fixes!

**Pull request merged**